model/lib/caching/state/state_type.dart

```diff
@@ -0,0 +1,21 @@
+
+import 'package:hive/hive.dart';
+import 'package:model/model.dart';
+
+part 'state_type.g.dart';
+
+@HiveType(typeId: CachingConstants.STATE_TYPE_IDENTIFY)
```

Comment:
IMO, all the Dao should be store in the `data` of mailbox. WDYT? @dab246

Reply:
ok